iocage_lib/ioc_start.py

```diff
@@ -590,6 +589,60 @@ def __start_jail__(self):
             "IOCAGE_NAME": f"ioc-{self.uuid}",
         }
 
+        if nat:
+            # We pass some environment variables to the shell script
+            # for nat based jails currently aiding in doing jail specific
+            # tasks in the host environment
```

Comment:
For 11.3, how about we add a big try/except here, just in case, to play it safe? Bad idea?

Reply:
There shouldn't be any issues but sure ;)